### PR TITLE
feat(ui5-calendar): implement horizon theme

### PR DESCRIPTION
**Pull request by Todor-ads**
feat(ui5-calendar): implement horizon theme
feat(ui5-DatePicker): implement horizon theme
feat(ui5-DateRangePicker): implement horizon theme
feat(ui5-DateTimePicker): implement horizon theme

### CODE REVIEW

**Comment by ilhan007** [comment]
/home/travis/build/SAP/ui5-webcomponents/packages/main/src/DayPicker.js
  268:50  error  Missing semicolon  semi
✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

File: packages/main/src/themes/DayPicker.css [hunk 1]
```diff
@@ -134,18 +134,28 @@
...
-	border: 0.125rem solid var(--sapLegend_CurrentDateTime);
+	border: var(--_ui5_daypicker_item_now_border);
 	padding: 0.0625rem;
 	box-shadow: inset 0 0 0 0.0625rem var(--_ui5_daypicker_item_selected_border_color);
+	outline: var(--_ui5_daypicker_item_now_outline);
+	outline-offset: var(--_ui5_daypicker_item_now_outline_offset);
+}
+
+.ui5-dp-item.ui5-dp-item--now.ui5-dp-item--selected {
+	/*--_ui5_daypicker_item_now_box_shadow*/
```
Comment (tsanislavgatev) [inline packages/main/src/themes/DayPicker.css:157]: remove commented code.

File: packages/main/src/themes/DayPicker.css [hunk 2]
```diff
@@ -224,3 +254,18 @@
...
+	outline-offset: -0.3125rem;
+	border: 2px solid var(--sapLegend_CurrentDateTime);
+	outline-color: var(--sapSelectedColor);
+}
+.ui5-dp-item.ui5-dp-item--now.ui5-dp-item--selected.ui5-dp-item--withsecondtype:focus::after{
+	outline-offset: -0.1875rem;
+	border-radius: 0.1875rem;
+}
+
+
```
Comment (tsanislavgatev) [inline packages/main/src/themes/DayPicker.css:268]: two empty lines, remove one of them

File: packages/main/src/themes/MonthPicker.css [hunk 3]
```diff
@@ -61,7 +63,8 @@
 }
 
 .ui5-mp-item.ui5-mp-item--selected:hover {
-	background-color: var(--sapContent_Selected_Background);
+	/*--_ui5_monthpicker_item_selected_hover_color*/
```
Comment (tsanislavgatev) [inline packages/main/src/themes/MonthPicker.css:66]: remove comment

File: packages/main/src/themes/WheelSlider.css [hunk 4]
```diff
@@ -60,6 +60,7 @@
 }
 
 .ui5-wheelslider-root .ui5-wheelslider-inner .ui5-wheelslider-item {
+	/*#0070f2;*/
```
Comment (tsanislavgatev) [inline packages/main/src/themes/WheelSlider.css:63]: remove comment

File: packages/main/src/themes/WheelSlider.css [hunk 5]
```diff
@@ -156,7 +157,8 @@
 :host([expanded]) .ui5-wheelslider-inner .ui5-wheelslider-item {
 	background: var(--_ui5_wheelslider_item_background_color);
 	color: var(--_ui5_wheelslider_item_text_color);
-	border: 1px solid var(--_ui5_wheelslider_item_border_color);
+	/*--_ui5_wheelslider_item_border*/
```
Comment (tsanislavgatev) [inline packages/main/src/themes/WheelSlider.css:160]: remove comment

File: packages/main/src/themes/YearPicker.css [hunk 6]
```diff
@@ -68,7 +70,8 @@
 }
 
 .ui5-yp-item.ui5-yp-item--selected:hover {
-	background-color: var(--_ui5_yearpicker_item_selected_focus);
+
```
Comment (tsanislavgatev) [inline packages/main/src/themes/YearPicker.css:73]: remove empty line

File: packages/main/src/themes/base/DayPicker-parameters.css [hunk 7]
```diff
@@ -13,10 +13,28 @@
...
+	--_ui5_daypicker_item_now_border: 0.125rem solid var(--sapLegend_CurrentDateTime);
+	--_ui5_daypicker_item_now_outline:none;
+	--_ui5_daypicker_item_now_outline_offset:none;
+	--_ui5_daypicker_item_now_outline_offset_focus_after: var(--_ui5_daypicker_item_now_outline_offset);
+	--_ui5_daypicker_item_selected_between_border: 5%;
+	--_ui5_daypicker_item_selected_between_background: transparent;
+	--_ui5_daypicker_item_selected_between_text_background: var(--sapList_SelectionBackgroundColor);
+	--_ui5_daypicker_item_selected_between_text_font: inherit;
+	--_ui5_daypicker_item_selected_between_hover_background: inherit;
+	--_ui5_daypicker_item_now_box_shadow:  inset 0 0 0 0.0625rem var(--_ui5_daypicker_item_selected_border_color);
```
Comment (tsanislavgatev) [inline packages/main/src/themes/base/DayPicker-parameters.css:38]: double space

File: packages/main/src/themes/base/DayPicker-parameters.css [hunk 8]
```diff
@@ -13,10 +13,28 @@
...
+	--_ui5_daypicker_item_selected_background: transparent;
+	--_ui5_daypicker_item_selected_box_shadow: inset 0 0 0 0.0625rem var(--sapContent_Selected_ForegroundColor),inset 0 0 0 0.0625rem var(--sapContent_Selected_ForegroundColor);
+	--_ui5_daypicker_item_selected_daytext_hover_background: transparent;
+	--_ui5_daypicker_item_outline_focus_after: none;
+	--_ui5_daypicker_item_border_radius_focus_after:none;
+	--_ui5_daypicker_item_border_focus_after: var(--_ui5_daypicker_item_outline_width) dotted var(--sapContent_FocusColor);
+	--_ui5_daypicker_item_width_focus_after: calc(100% - 0.25rem);
+	--_ui5_daypicker_item_height_focus_after: calc(100% - 0.25rem);
+	--_ui5_daypicker_item_now_border: 0.125rem solid var(--sapLegend_CurrentDateTime);
+	--_ui5_daypicker_item_now_outline:none;
```
Comment (tsanislavgatev) [inline packages/main/src/themes/base/DayPicker-parameters.css:30]: add space

File: packages/main/src/themes/base/DayPicker-parameters.css [hunk 9]
```diff
@@ -13,10 +13,28 @@
...
+	--_ui5_daypicker_item_now_outline_offset:none;
```
Comment (tsanislavgatev) [inline packages/main/src/themes/base/DayPicker-parameters.css:31]: add space

File: packages/main/src/themes/base/MonthPicker-parameters.css [hunk 10]
```diff
@@ -6,4 +6,11 @@
 	--_ui5_monthpicker_item_focus_after_height: calc(100% - 0.375rem);
 	--_ui5_monthpicker_item_focus_after_border: 1px dotted var(--sapContent_FocusColor);
 	--_ui5_monthpicker_item_focus_after_offset: 2px;
+	--_ui5_monthpicker_item_focus_after_border_radius:0;
+	--_ui5_monthpicker_item_selected_text_color: var(--sapContent_ContrastTextColor);
+	--_ui5_monthpicker_item_selected_background_color:var(--sapSelectedColor);
```
Comment (tsanislavgatev) [inline packages/main/src/themes/base/MonthPicker-parameters.css:11]: add space

File: packages/main/src/themes/base/MonthPicker-parameters.css [hunk 11]
```diff
@@ -6,4 +6,11 @@
 	--_ui5_monthpicker_item_focus_after_height: calc(100% - 0.375rem);
 	--_ui5_monthpicker_item_focus_after_border: 1px dotted var(--sapContent_FocusColor);
 	--_ui5_monthpicker_item_focus_after_offset: 2px;
+	--_ui5_monthpicker_item_focus_after_border_radius:0;
```
Comment (tsanislavgatev) [inline packages/main/src/themes/base/MonthPicker-parameters.css:9]: add space

File: packages/main/src/themes/base/YearPicker-parameters.css [hunk 12]
```diff
@@ -1,10 +1,16 @@
 :root {
 	--_ui5_yearpicker_item_selected_focus: var(--sapContent_Selected_Background);
+	--_ui5_yearpicker_item_selected_hover_color:var(--sapContent_Selected_Background);
```
Comment (tsanislavgatev) [inline packages/main/src/themes/base/YearPicker-parameters.css:3]: add space

File: packages/main/src/themes/sap_horizon/CalendarHeader-parameters.css [hunk 13]
```diff
@@ -5,4 +5,15 @@
 	--_ui5_calendar_header_middle_button_focus_after_height: calc(100% - 0.375rem);
 	--_ui5_calendar_header_middle_button_focus_after_top_offset: 0.125rem;
 	--_ui5_calendar_header_middle_button_focus_after_left_offset: 0.125rem;
+	--_ui5_calendar_header_arrow_button_border: 0.0625rem solid var(--sapList_Hover_Background);
+	--_ui5_calendar_header_arrow_button_border_radius: 0.5rem;
+	--_ui5_calendar_header_button_background_color: var(--sapList_Hover_Background);
+	--_ui5_calendar_header_arrow_button_box_shadow: 0 0 0.125rem 0 rgb(85 107 130 / 72%);
+	--_ui5_calendar_header_middle_button_focus_border_radius: 0.5rem;
+	--_ui5_calendar_header_middle_button_focus_border: none;
+	--_ui5_calendar_header_middle_button_focus_after_border:none;
```
Comment (tsanislavgatev) [inline packages/main/src/themes/sap_horizon/CalendarHeader-parameters.css:14]: add space

File: packages/main/src/themes/sap_horizon/DayPicker-parameters.css [hunk 14]
```diff
@@ -3,4 +3,29 @@
 :root {
 	--_ui5_daypicker_item_border_radius: 0.25rem;
 	--_ui5_daypicker_item_now_inner_border_radius: 0.125rem;
+	--_ui5_daypicker_item_border_radius: 0.5rem;
+	--_ui5_daypicker_item_selected_background: transparent;
+	--_ui5_daypicker_item_selected_box_shadow: inset 0 0 0 0.125rem var(--sapContent_Selected_ForegroundColor);
+	--_ui5_daypicker_item_selected_daytext_hover_background: transparent;
+	--_ui5_daypicker_item_outline_focus_after: 0.125rem solid var(--sapContent_Selected_ForegroundColor);
+	--_ui5_daypicker_item_border_radius_focus_after: 0.5rem;
+	--_ui5_daypicker_item_border_focus_after:none;
```
Comment (tsanislavgatev) [inline packages/main/src/themes/sap_horizon/DayPicker-parameters.css:12]: add space

File: packages/main/src/themes/base/DayPicker-parameters.css [hunk 15]
```diff
@@ -13,10 +13,28 @@
...
 	--_ui5_daypicker_item_outline_width: 1px;
 	--_ui5_daypicker_item_outline_offset: 1px;
-	--_ui5_daypicker_item_now_outline_offset: 1px;
 	--_ui5_daypicker_item_now_selected_outline_offset: 2px;
 	--_ui5_daypicker_item_now_focus_after_width: calc(100% - .25rem);
 	--_ui5_daypicker_item_now_focus_after_height: calc(100% - .25rem);
 	--_ui5_daypicker_item_now_selected_focus_after_width: calc(100% - .375rem);
 	--_ui5_daypicker_item_now_selected_focus_after_height: calc(100% - .375rem);
+	--_ui5_daypicker_item_selected_background: transparent;
+	--_ui5_daypicker_item_selected_box_shadow: inset 0 0 0 0.0625rem var(--sapContent_Selected_ForegroundColor),inset 0 0 0 0.0625rem var(--sapContent_Selected_ForegroundColor);
```
Comment (tsanislavgatev) [inline packages/main/src/themes/base/DayPicker-parameters.css:22]: can we simplify this line?

File: packages/main/src/themes/sap_horizon/MonthPicker-parameters.css [hunk 16]
```diff
@@ -2,4 +2,16 @@
...
+	--_ui5_monthpicker_item_focus_after_border: none;
+	--_ui5_monthpicker_item_focus_after_border_radius: 0.25rem;
+	--_ui5_monthpicker_item_focus_after_width: calc(100% - 0.5rem);
+	--_ui5_monthpicker_item_focus_after_height: calc(100% - 0.5rem);
+	--_ui5_monthpicker_item_focus_after_offset: 0.25rem;
+	--_ui5_monthpicker_item_selected_text_color: var(--sapContent_Selected_TextColor);
+	--_ui5_monthpicker_item_selected_background_color:var(--sapLegend_WorkingBackground);
+	--_ui5_monthpicker_item_selected_hover_color: var(--sapList_Hover_Background);
+	--_ui5_monthpicker_item_selected_box_shadow: inset 0 0 0 0.15rem var(--sapSelectedColor);
+	--_ui5_monthpicker_item_focus_after_outline:  0.125rem  solid var(--sapSelectedColor);
```
Comment (tsanislavgatev) [inline packages/main/src/themes/sap_horizon/MonthPicker-parameters.css:15]: two spaces

File: packages/main/src/themes/sap_horizon/WheelSlider-parameters.css [hunk 17]
```diff
@@ -13,4 +13,10 @@
 	--_ui5_wheelslider_active_item_text_color: var(--sapContent_Selected_TextColor);
 	--_ui5_wheelslider_selection_frame_color: var(--sapList_SelectionBorderColor);
 	--_ui_wheelslider_item_border_radius: var(--_ui5_button_border_radius);
+	--_ui5_wheelslider_collapsed_item_text_color: #0070f2;
```
Comment (tsanislavgatev) [inline packages/main/src/themes/sap_horizon/WheelSlider-parameters.css:16]: Can we use parameter for this?

File: packages/main/src/themes/sap_horizon/WheelSlider-parameters.css [hunk 18]
```diff
@@ -13,4 +13,10 @@
...
+	--_ui5_wheelslider_item_border_color: #0070f2;
```
Comment (tsanislavgatev) [inline packages/main/src/themes/sap_horizon/WheelSlider-parameters.css:17]: Can we use parameter for this?

File: packages/main/src/themes/sap_horizon/YearPicker-parameters.css [hunk 19]
```diff
@@ -2,4 +2,15 @@
 
 :root {
 	--_ui5_yearpicker_item_border_radius: 0.25rem;
+	--_ui5_yearpicker_item_border_radius:0.5rem;
```
Comment (tsanislavgatev) [inline packages/main/src/themes/sap_horizon/YearPicker-parameters.css:5]: add space after semicolon

File: packages/main/src/themes/base/CalendarHeader-parameters.css [hunk 20]
```diff
@@ -10,4 +10,11 @@
 	--_ui5_calendar_header_middle_button_focus_after_height: calc(100% - 0.25rem);
 	--_ui5_calendar_header_middle_button_focus_after_top_offset: 1px;
 	--_ui5_calendar_header_middle_button_focus_after_left_offset: 1px;
+	--_ui5_calendar_header_button_background_color: none;
+	--_ui5_calendar_header_arrow_button_box_shadow: none;
+	--_ui5_calendar_header_middle_button_focus_background: transparent;
+	--_ui5_calendar_header_middle_button_focus_outline:none;
```
Comment (tsanislavgatev) [inline packages/main/src/themes/base/CalendarHeader-parameters.css:16]: add space after semicolon

File: packages/main/src/themes/DayPicker.css [hunk 21]
```diff
@@ -202,15 +225,22 @@
...
+	background-color: var(--_ui5_daypicker_item_selected_between_text_background);
 	color: var(--sapTextColor);
+	font-weight: var(--_ui5_daypicker_item_selected_between_text_font);
 }
 
 .ui5-dp-item.ui5-dp-item--selected-between,
 .ui5-dp-item[hovered] {
 	border: 1px solid var(--sapContent_Selected_Background);
-	border-radius: 5%;
+	border-radius:var(--_ui5_daypicker_item_selected_between_border);
```
Comment (tsanislavgatev) [inline packages/main/src/themes/DayPicker.css:237]: add space after semicolon

**Review by tsanislavgatev — CHANGES_REQUESTED** [review]

File: packages/main/src/themes/DayPicker.css (hunk 1 above)
Comment (Todor-ads) [reply packages/main/src/themes/DayPicker.css:157]: done

File: packages/main/src/themes/DayPicker.css (hunk 2 above)
Comment (Todor-ads) [reply packages/main/src/themes/DayPicker.css:268]: done

File: packages/main/src/themes/MonthPicker.css (hunk 3 above)
Comment (Todor-ads) [reply packages/main/src/themes/MonthPicker.css:66]: done

File: packages/main/src/themes/YearPicker.css (hunk 6 above)
Comment (Todor-ads) [reply packages/main/src/themes/YearPicker.css:73]: done

File: packages/main/src/themes/base/DayPicker-parameters.css (hunk 7 above)
Comment (Todor-ads) [reply packages/main/src/themes/base/DayPicker-parameters.css:38]: done

File: packages/main/src/themes/base/DayPicker-parameters.css (hunk 8 above)
Comment (Todor-ads) [reply packages/main/src/themes/base/DayPicker-parameters.css:30]: done

File: packages/main/src/themes/base/DayPicker-parameters.css (hunk 9 above)
Comment (Todor-ads) [reply packages/main/src/themes/base/DayPicker-parameters.css:31]: done

File: packages/main/src/themes/base/MonthPicker-parameters.css (hunk 10 above)
Comment (Todor-ads) [reply packages/main/src/themes/base/MonthPicker-parameters.css:11]: done

File: packages/main/src/themes/base/MonthPicker-parameters.css (hunk 11 above)
Comment (Todor-ads) [reply packages/main/src/themes/base/MonthPicker-parameters.css:9]: done

File: packages/main/src/themes/base/YearPicker-parameters.css (hunk 12 above)
Comment (Todor-ads) [reply packages/main/src/themes/base/YearPicker-parameters.css:3]: done

File: packages/main/src/themes/sap_horizon/CalendarHeader-parameters.css (hunk 13 above)
Comment (Todor-ads) [reply packages/main/src/themes/sap_horizon/CalendarHeader-parameters.css:14]: done

File: packages/main/src/themes/base/DayPicker-parameters.css (hunk 15 above)
Comment (Todor-ads) [reply packages/main/src/themes/base/DayPicker-parameters.css:22]: done

File: packages/main/src/themes/sap_horizon/MonthPicker-parameters.css (hunk 16 above)
Comment (Todor-ads) [reply packages/main/src/themes/sap_horizon/MonthPicker-parameters.css:15]: done

File: packages/main/src/themes/sap_horizon/YearPicker-parameters.css (hunk 19 above)
Comment (Todor-ads) [reply packages/main/src/themes/sap_horizon/YearPicker-parameters.css:5]: done

File: packages/main/src/themes/base/CalendarHeader-parameters.css (hunk 20 above)
Comment (Todor-ads) [reply packages/main/src/themes/base/CalendarHeader-parameters.css:16]: done

File: packages/main/src/themes/DayPicker.css (hunk 21 above)
Comment (Todor-ads) [reply packages/main/src/themes/DayPicker.css:237]: done

File: packages/main/src/themes/sap_horizon/WheelSlider-parameters.css (hunk 18 above)
Comment (Todor-ads) [reply packages/main/src/themes/sap_horizon/WheelSlider-parameters.css:17]: remove changes on WheelSlider

File: packages/main/src/themes/sap_horizon/WheelSlider-parameters.css (hunk 17 above)
Comment (Todor-ads) [reply packages/main/src/themes/sap_horizon/WheelSlider-parameters.css:16]: remove changes on WheelSlider

File: packages/main/src/themes/WheelSlider.css (hunk 5 above)
Comment (Todor-ads) [reply packages/main/src/themes/WheelSlider.css:160]: remove changes on WheelSlider

File: packages/main/src/themes/WheelSlider.css (hunk 4 above)
Comment (Todor-ads) [reply packages/main/src/themes/WheelSlider.css:63]: remove changes on WheelSlider

**Pull request merged**